app/services/actions/foreman/puppet_class/import.rb

```diff
@@ -15,9 +15,9 @@ def humanized_output
           return nil if input[:changed].nil?
 
           humanized_output = []
-          humanized_output << _('Add') + ' ' + format_env_and_classes_input(input[:changed][:new]) if input[:changed][:new].present?
-          humanized_output << _('Remove') + ' ' + format_env_and_classes_input(input[:changed][:obsolete]) if input[:changed][:obsolete].present?
-          humanized_output << _('Update') + ' ' + format_env_and_classes_input(input[:changed][:updated]) if input[:changed][:updated].present?
+          humanized_output << _('Add %s') % format_env_and_classes_input(input[:changed][:new]) if input[:changed][:new].present?
```

Comment:
The verb should still be part of the string _inside_ the method as it's part of the same sentence - it isn't a separate sentence or fragment, i.e. inside `format_env_and_classes_input`:

    _("Add environment %s")

and append versions without the verb:

    _("environment %s")

unless you rewrite it entirely into separate fragments or sentences, which it isn't right now.